ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -116,6 +128,18 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-mini-perm-arm-f28
+  cron: 54 19 15 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
```

Comment:
Please help remove below `aws-ipi-ovn-ipsec-arm-mixarch-day2-f28`, it's duplicate to `aws-ipi-ovn-ipsec-arm-mixarch-f2`